[2024-09-08T00:00:01.047+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-08T00:00:01.074+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v3.first_task scheduled__2024-09-07T00:00:00+00:00 [queued]>
[2024-09-08T00:00:01.097+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v3.first_task scheduled__2024-09-07T00:00:00+00:00 [queued]>
[2024-09-08T00:00:01.104+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-08T00:00:01.125+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): first_task> on 2024-09-07 00:00:00+00:00
[2024-09-08T00:00:01.202+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'first_task', 'scheduled__2024-09-07T00:00:00+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmphge17bhb']
[2024-09-08T00:00:01.230+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=61735) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-08T00:00:01.268+0000] {standard_task_runner.py:105} INFO - Job 212: Subtask first_task
[2024-09-08T00:00:01.269+0000] {standard_task_runner.py:72} INFO - Started process 61741 to run task
[2024-09-08T00:00:01.560+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_v3.first_task scheduled__2024-09-07T00:00:00+00:00 [running]> on host 626f2e28dec2
[2024-09-08T00:00:01.877+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='my_first_dag_v3' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-07T00:00:00+00:00'
[2024-09-08T00:00:01.888+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-08T00:00:01.963+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-08T00:00:01.968+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-09-08T00:00:02.009+0000] {subprocess.py:86} INFO - Output:
[2024-09-08T00:00:02.021+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-09-08T00:00:02.024+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-08T00:00:02.062+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-08T00:00:02.065+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=first_task, run_id=scheduled__2024-09-07T00:00:00+00:00, execution_date=20240907T000000, start_date=20240908T000001, end_date=20240908T000002
[2024-09-08T00:00:02.116+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-08T00:00:02.226+0000] {taskinstance.py:3891} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-09-08T00:00:02.241+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
